src/emit/binary.rs

```diff
@@ -868,10 +868,34 @@ impl<'a> Binary<'a> {
     pub(crate) fn llvm_type(&self, ty: &Type, ns: &Namespace) -> BasicTypeEnum<'a> {
         if ty.is_builtin_struct() == Some(StructType::AccountInfo) {
             return self
-                .module
-                .get_struct_type("struct.SolAccountInfo")
-                .unwrap()
-                .into();
+                .context
+                .struct_type(
+                    &[
+                        self.context
+                            .i8_type()
+                            .ptr_type(AddressSpace::default())
+                            .as_basic_type_enum(), // SolPubkey *
```

Comment:
`byte_ptr!()`?

Reply:
Well, any type of pointer works, because they are opaque.

Reply:
The pointer here is pointing to `SolPubkey`, but I did not want to insert the struct definition just to retrieve a pointer to it.

Reply:
I'm just suggesting to replace `self.context
                            .i8_type()
                            .ptr_type(AddressSpace::default())
                            .as_basic_type_enum()` with `byte_ptr!()`